src/index.d.ts

```diff
@@ -834,6 +834,14 @@ declare namespace admin.remoteConfig {
      */
     parameters: { [key: string]: RemoteConfigParameter };
 
+    /**
+     * Map of parameter group names to their parameter group objects.
+     * A group's name is mutable but must be unique among groups in the config.
```

Comment:
Not sure, but where we say "the config" here, do we mean the Remote Config template?  Or maybe the `RemoteConfig` object, as you mention it in line 891?

Reply:
Good catch. It should be `Remote Config template`. Updated in both places. Thanks!